### DISCUSSION TITLE
Emulating Vim Mode Plus' occurrence modifier

### DISCUSSION THREAD

**Discussion by bennypowers** [mg979/vim-visual-multi#200] [Q&A]
Hello!

The good old atom plugin [vim-mode-plus](https://github.com/t9md/atom-vim-mode-plus) by @t9md has a killer feature called the [occurrence modifier](https://github.com/t9md/atom-vim-mode-plus/wiki/OccurrenceModifier) which is very similar to things this plugin can do

`c o i p` - change occurrence inner paragraph

[![COIP](https://github.com/t9md/t9md/raw/6f53558743f39cf4b8d181b056222281f05b3161/img/vim-mode-plus/v0.55.0/cip-vs-coip.gif?raw=true)]

How could I go about constructing bindings like these with your plugin?

**Comment by mg979** [comment]
Try `:h vm-find-operator`

It requires a couple more keystrokes, tbh.

**Reply by bennypowers** [reply]
couldn't i setup a textobject or a binding that automatically entered multimode for `m` (or `o`)?

**Comment by IndianBoy42** [comment]
have you figured out or tried to setup that binding?
